public/app/js/chatroomPlus.js

```diff
@@ -66,15 +66,17 @@ function toggleChatRoom(e) {
     var percentage = (e.pageX / window.innerWidth) * 100;
     if(percentage < 70){
         $('#editor-frame').css("width", "98%");
-        $('#editorframe').css("width", "calc(100% - 5px)"); // subtract the length of the dragbar
+        $('#editorframe').css("width", "100%");
```

Comment:
We need to find a way to rename these two elements, when I was developing, this is super confusing.
`#editor-frame` and `#editorframe`

Reply:
true this needs to be refactored